utils/selinux-python/Makefile

```diff
@@ -0,0 +1,157 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=selinux-python
+PKG_VERSION:=3.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/SELinuxProject/selinux/releases/download/20200710
+PKG_HASH:=f4d0a1a030bc291a6af498b26e0676b745075dd289a8ba16cdec86c3ea8f2f02
+PKG_BUILD_DEPENDS:=PACKAGE_selinux-audit2allow:libsepol
```

Comment:
Now that I look at it again that `PKG_BUILD_DEPENDS` is a bit strange. `libselinux` depends on `libsepol`, and `python- libselinux` depends on that. So `libsepol` should anyway always become an indirect build dependency...? But maybe I'm getting something wrong here...

Reply:
I am running a build test to look into this. More to follow tomorrow morning, I hope.

Reply:
libselinux indeed depends on libsepol AFAIK. So libsepol essentially is the most fundamental dependency.

Reply:
The selinux-python packages build without the PKG_BUILD_DEPENDS, so I removed it.